src/sender_p.h

```diff
@@ -66,7 +66,8 @@ class SenderPrivate
     int sendMessageTimeout = 60000;
 
     QByteArray responseText;
-    int responseCode;
+    int responseCode = -1;
```

Comment:
I wonder why you added a responseCode here and no other file mentions it.

Reply:
This member variable is uninitialized until first response is processed. Until then this member contains any value. This way when you access this value and it is equal -1 you know that there was no response processed yet.